src/util/version.ts

```diff
@@ -57,11 +57,13 @@ export async function printVersion(
 
 export function variants(version: string) {
   const kabobBuild = version.replace(/[+]/g, "-");
+  const semver = parse(version);
   // todo: add any other platform specific variants here.
   return {
     version_default: version,
     version_dotnet: kabobBuild,
     version_docker: kabobBuild,
+    version_major: `v${semver.major}`,
```

Comment:
What do you think about adding a separate `prefix: v` field?

So then your replacement could be:

```
  kind: regexp
  pattern: ...
  format: major
  prefix: v
```

Reply:
Then we could prefix any of the other formats with v also if desired.